### ISSUE TITLE
Uncaught TypeError: Cannot read property 'fromString' of undefined 

### ISSUE THREAD

**Issue by francoisauclair911** [peterbraden/ical.js#92]
Hitting these errors when trying to run the basic example 

```
node-ical.js?38f9:34 Uncaught TypeError: Cannot read property 'fromString' of undefined
    at Object.ical.objectHandlers.END (node-ical.js?38f9:34)
    at Object.handleObject (ical.js?0ad9:253)
    at Object.parseICS (ical.js?0ad9:294)
    at Request.eval [as _callback] (node-ical.js?38f9:11)
    at Request.self.callback (request.js?4b05:199)
    at Request.EventEmitter.emit (events.js?faa1:84)
    at Request.eval (request.js?4b05:1036)
    at Request.EventEmitter.emit (events.js?faa1:96)
    at exports.IncomingMessage.eval (request.js?4b05:963)
    at exports.IncomingMessage.EventEmitter.emit (events.js?faa1:96)
```

**Comment by scazzy** [comment]
I faced this too. pretty weird it didn't cover basic .ics rules.

@francoisauclair911 when did you break for you? with multi-day or recurring events?

**Comment by pavermakov** [comment]
I have the same issue

<img width="737" alt="Снимок экрана 2019-12-11 в 19 19 47" src="https://user-images.githubusercontent.com/10779996/70639384-750afa00-1c4b-11ea-8cde-fdfbf5865582.png">

**Comment by pavermakov** [comment]
Any ideas how to fix it?

**Comment by pavermakov** [comment]
Solved by upgrading to 0.6.0
